src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -133,6 +148,7 @@ def __init__(
         self._variant_id = variant_id
         self._validate_inputs = validate_inputs
         self._worker_count = flow_executor._worker_count
+        self.batch_run_start_method = os.environ.get("PF_BATCH_METHOD")
         use_fork = multiprocessing.get_start_method() == "fork"
```

Comment:
It seems that here we checked the global start method instead of the method in the environment.